### PR TITLE
Development

### PR DESCRIPTION
**Pull request by DariusDangi**
Ready for first code review.

### CODE REVIEW

File: DataLayer/DetermineAnimalType.xaml
Comment (RobertFWF) [inline DataLayer/DetermineAnimalType.xaml:1]: Coloana de interes pentru "Animal" poate fi in Config

File: Exceptions_Screenshots/ExceptionScreenshot_240814.104822.png
Comment (RobertFWF) [inline Exceptions_Screenshots/ExceptionScreenshot_240814.104822.png:1]: Sterge screenshot-urile

File: Framework/CloseAllApplications.xaml
Comment (RobertFWF) [inline Framework/CloseAllApplications.xaml:1]: Nefolosit

File: Framework/GetTransactionData.xaml
Comment (RobertFWF) [inline Framework/GetTransactionData.xaml:1]: io_dt_TransactionData nefolosit deci il poti sterge de peste tot

File: Framework/InitAllApplications.xaml
Comment (RobertFWF) [inline Framework/InitAllApplications.xaml:1]: Sheet-ul in Config

File: Framework/KillAllProcesses.xaml
Comment (RobertFWF) [inline Framework/KillAllProcesses.xaml:1]: Nefolosit

File: Framework/SetTransactionStatus.xaml
Comment (RobertFWF) [inline Framework/SetTransactionStatus.xaml:1]: Statusul la SE e hardcoded, pune-l cu in_SystemException.Message
Nu folosesti CloseAllApplication deci poti sa il stergi si din proiect si de peste tot unde este folosit, la fel ca celelalte workflow-uri nefolosite.

La Write Status pune coloanele de interes si sheet-ul in Config 

Verifica numele activitatilor default

File: Main.xaml
Comment (RobertFWF) [inline Main.xaml:1]: In EndProcess ai activitatea de Send Output Report care se poate pune intr-un workflow separat tot in ServiceLayer.

To Subject Body etc in Config

File: ServiceLayer/CallAnimalAPI.xaml
Comment (RobertFWF) [inline ServiceLayer/CallAnimalAPI.xaml:1]: Dupa fiecare call de API adauga un if sa verifici statusul, daca nu a avut succes api-ul nu are rost sa continue workflow-ul
image

Endpoint-urile la API-uri in Config

File: ServiceLayer/SendEmailImageAttachment.xaml
Comment (RobertFWF) [inline ServiceLayer/SendEmailImageAttachment.xaml:1]: Adresa la care trimiti sa fie in Config

**Review by RobertFWF — COMMENTED** [review]
First Code Review

File: Framework/GetTransactionData.xaml
Comment (FlorinBarsan) [inline Framework/GetTransactionData.xaml:1]: remove variables that have no use (transaction fields) from the whole project

File: .codedworkflows/ConnectionsFactory.cs
Comment (FlorinBarsan) [inline .codedworkflows/ConnectionsFactory.cs:1]: delete that test.txt file from the project

**Review by FlorinBarsan — CHANGES_REQUESTED** [review]
just a few changes left, implement them please and request a re-review

File: Framework/GetTransactionData.xaml
Comment (DariusDangi) [reply Framework/GetTransactionData.xaml:1]: Removed the variables/arguments transactionField1 and transactionField2 from the whole project

File: .codedworkflows/ConnectionsFactory.cs
Comment (DariusDangi) [reply .codedworkflows/ConnectionsFactory.cs:1]: deleted the test.txt file from project

**Review by FlorinBarsan — APPROVED** [review]
good job!

**Pull request merged**